youtube_dl/extractor/laola1tv.py

```diff
@@ -54,16 +89,17 @@ def _real_extract(self, url):
 
         title = xpath_text(hd_doc, './/video/title', fatal=True)
         flash_url = xpath_text(hd_doc, './/video/url', fatal=True)
-        uploader = xpath_text(hd_doc, './/video/meta_organistation')
+        uploader = xpath_text(hd_doc, './/video/meta_organisation')
         is_live = xpath_text(hd_doc, './/video/islive') == 'true'
 
         categories = xpath_text(hd_doc, './/video/meta_sports')
         if categories:
             categories = categories.split(',')
 
-        ident = random.randint(10000000, 99999999)
+        timestamp = time.strftime('%Y%m%d%H%M%S', time.gmtime())
+        ident = '%d%s' % (random.randint(10000000, 99999999), timestamp)
```

Comment:
I appreciate your assistance, but I still don't understand how I can get the timestamp from the URL you gave.